Home work 18 Sprint ( Fragments )

Evgen1994-McD opened the pull request.

Evgen1994-McD committed:
Активити медиа

Evgen1994-McD committed:
сделал оба фрагмента

Evgen1994-McD committed:
update

Evgen1994-McD committed:
Написал Активити под фрагменты, Вью модел для активити.
В процессе написания фрагментов

Evgen1994-McD committed:
Дописал фрагмент фаворит треков, написал адаптер для вью пейджера, осталось написать один фрагмент

Evgen1994-McD committed:
апдейти

Evgen1994-McD committed:
Настроил оба фрагмента, таблайоут, медиатор, текст, разметка

Evgen1994-McD committed:
апдейт

Evgen1994-McD committed:
сохранение состояния таба

Evgen1994-McD committed:
перевел все зависимости в версион - каталог для удобства по совету ревьюера ( libs.versions.toml )

Evgen1994-McD committed:
ошибся - передавал мутабл лайв дата на фрагменты ( исправил)

anastasianadelshtehel commented:
верстка не соответствует макетам:
![5368403065381713213](https://github.com/user-attachments/assets/1d0e095f-76be-4298-a9cd-46acd61abfbd)
![5368403065381713214](https://github.com/user-attachments/assets/c6b0efa1-08b5-4b20-bde1-e059677bba8e)

Evgen1994-McD committed:
1. Исправил названия ( жирный шрифт)
2. Касательно переключения вкладок при нажатии ( не только по свайп) - всё работает. Таблайоут медиатор настроен, и переключение происходит и так и так.

Evgen1994-McD committed:
Нашел проблему, исправил отступы, добавил слушатель на всякий случай. Проблема была в кривой разметке, после устранения разметки всё встало и нажимается нормально.
Так же устранил остальные замечания, проверил на всех видах экранов ( мал + сред + большой)
